utils/check/handle_rpms.sh

```diff
@@ -38,16 +38,53 @@ if ! grep "gpgcheck = false" "${repo_file}"; then
   echo "gpgcheck = false" | sudo tee -a "${repo_file}"
 fi
 
-sudo yum -y remove cart cart-debuginfo cart-devel cart-tests \
-                   daos daos-client daos-debuginfo daos-devel daos-server \
-                   daos-tests CUnit dpdk fio fuse fuse-lib hwloc \
-                   libabt0 libcmocka libfabric libfabric-devel libisa-l \
-                   libisa-l_crypto libopa1 libpmem libpmemblk mercury ndctl\
-                   ompi openmpi3 openpa-devel \
-                   pexpect protobuf-c pmix \
-                   python-configshell python-urwid \
-                   raft raft-debuginfo raft-devel \
-                   spdk spdk-tools
+sudo yum -y remove cart \
+                   cart-debuginfo \
+                   cart-devel \
+                   cart-tests \
```

Comment:
Cart is no longer in separate RPMs.  If you need to re-spin this PR, this seems like as good as any to remove these from here.

Reply:
Sounds good.